src/components/SponsorPaymentModal.vue

```diff
@@ -10,7 +10,7 @@
         </div>
       </div>
       <div class="columns">
-        <div class="column content is-hidden-mobile">
+        <div class="column content is-hidden-mobile" v-if="!isPhone">
```

Comment:
`is-hidden-mobile` is enough to hide in mobile, why ` v-if="!isPhone"`?

Reply:
nope in my Windows Chrome

Reply:
Can't catch your point. Your windows Chrome of course won't hide.. You are desktop 🤣

Reply:
![image](https://user-images.githubusercontent.com/44204108/47855121-9e3d4200-de1e-11e8-83c6-01e843d784ab.png)

You need to click this to simulate a mobile device (UA + Correct responsive screen size)